cob/cob.c

```diff
@@ -877,6 +1034,70 @@ static int cob_ns_lookup(struct m0_cob *cob);
 static int cob_oi_lookup(struct m0_cob *cob);
 static int cob_fab_lookup(struct m0_cob *cob);
 
+M0_INTERNAL int m0_cob_bc_lookup(struct m0_cob *cob,
+				 struct m0_cob_bckey *bc_key,
+				 struct m0_cob_bcrec *bc_rec)
+{
+	struct m0_buf key;
+	struct m0_buf val;
+	int           rc;
+
+	M0_PRE(bc_key != NULL &&
+	       m0_fid_is_set(&bc_key->cbk_pfid));
+
+	m0_buf_init(&key, bc_key, m0_cob_bckey_size());
+	m0_buf_init(&val, bc_rec, m0_cob_bcrec_size());
+
+	rc = cob_table_lookup(&cob->co_dom->cd_bytecount, &key, &val);
+	if (rc == 0) {
+		cob->co_flags |= M0_CA_BCREC;
+	}
```

Comment:
Remove brackets  for single line if statement.

Reply:
Done.